### ISSUE TITLE
[Bug Report] JDK 17 doesn’t work

### ISSUE THREAD

**Issue by CyberFlameGO** [jagrosh/MusicBot#913]
### Bug Description

The bot doesn't do anything past loading config

### Steps to Reproduce

1. Try to use JDK 17
2. Observe that it doesn't work

### Expected Result

I expected the bot to work

### Debug Output

```shell
N/A, 0.3.5, JDK 17 (tried with Oracle and AWS Corretto)
```

### Additional Info

JDK 16 does though

### Checklist

- [X] I have looked for information about this [within the documentation](https://jmusicbot.com/)
- [X] I have searched for similar issues [on the issues page](https://github.com/jagrosh/MusicBot/issues?q=is%3Aissue)
- [X] I am running the latest version of the bot: ![Release](https://img.shields.io/github/release/jagrosh/MusicBot.svg)

**Comment by Sanduhr32** [comment]
Wdym? Do you have any logs for whats happening?

**Comment by CyberFlameGO** [comment]
Nothing apart from a message saying the config loaded. The verbose flag isn't useful either

**Comment by Sanduhr32** [comment]
Honestly, it works for me on an older version of JMusicBot, so old that lavaplayer can't play anything lol

**Comment by CyberFlameGO** [comment]
What version/commit?

**Comment by MichailiK** [comment]
Cannot reproduce using OpenJDK 17.

**Comment by Sanduhr32** [comment]
> What version/commit?

Whatever it was 27th september 2020

**Comment by CyberFlameGO** [comment]
From https://github.com/jagrosh/MusicBot/issues/913#issuecomment-927100284 by @MichailiK 
> Cannot reproduce using Oracle OpenJDK 17.

What platform did you try this on? I forgot to note that the platform I experienced this bug on is Debian 11 (bullseye)

`cat /etc/os-release`
Output: 
```
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

**Comment by Un0Du0** [comment]
I can confirm I have the same issue with Ubuntu 20.04 LTS.
Running Ubuntu 20.04.3 LTS
>java --version
openjdk 17 2021-09-14
OpenJDK Runtime Environment (build 17+35-Ubuntu-120.04)
OpenJDK 64-Bit Server VM (build 17+35-Ubuntu-120.04, mixed mode, sharing)

> java -Dnogui=true -jar JMusicBot-0.3.5.jar
[13:34:08] [INFO] [Startup]: Loaded config from /home/discord-music/config.txt


And it just sits there, won't respond in Discord.

If I use openjdk 16 everything works fine.

**Comment by Sanduhr32** [comment]
> `cat /etc/os-release`
> Output: 
> ```
> PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
> NAME="Debian GNU/Linux"
> VERSION_ID="11"
> VERSION="11 (bullseye)"
> VERSION_CODENAME=bullseye
> ID=debian
> HOME_URL="https://www.debian.org/"
> SUPPORT_URL="https://www.debian.org/support"
> BUG_REPORT_URL="https://bugs.debian.org/"
> ```

I'm also on Debian 11 Bullseye and have no issues, including Win10/Windows with Java 17, tho I stopped using Oracle/Open JDK a few years ago.
I use the Azul Zulu JDK builds, they are my preference, because they have higher quality standards and most issues fixed.

**Comment by giovanniwijaya** [comment]
Cannot reproduce using OpenJDK 17 GA, both on Windows 11 and Debian 11. But I remember encountering the same issue a few months ago when JDK 17 was still in early access.

**Comment by CyberFlameGO** [comment]
The errors encountered are on GA builds (from apt)

**Comment by CyberFlameGO** [comment]
Seems to be fixed, I also tried it on Coretto (not shown here)
![Screen Shot 2021-11-15 at 2 58 02 PM](https://user-images.githubusercontent.com/24910512/141710468-34bd8708-d665-4e11-837b-cefb30e384cf.png)
I'll keep this issue open in case anyone else is still encountering the issue, unless I'm not supposed to do that, in which case a collaborator can close this

**Comment by tojatos** [comment]
I can reproduce this issue when running the bot with `image: openjdk:latest` in docker-compose on Arch Linux.
However, the same docker-compose is working on a different host.

The issue was resolved by downgrading the image to `openjdk:8`.

**Comment by nlutterman** [comment]
I rebuilt the project with Maven and OpenJDK 17 and things worked then. Perhaps the released `jar` is being built against an older SDK?

**Comment by CyberFlameGO** [comment]
It's targeting 1.8, and the errors I had previously were using a server
with only JDK 17, which is the server I compiled it on (still targeting 1.8)

**Comment by e00E** [comment]
I have the same issue using the current release JMusicBot-0.3.6.jar. Alpine Linux 3.15 openjdk17 package hangs after "Loaded Config". openjdk16 package works.

**Comment by Un0Du0** [comment]
This appears to be fixed with version 0.3.8

**Comment by benthillerkus** [comment]
Cannot reproduce on Openjdk 18 on Ubuntu 22
